.github/workflows/deploy.yml

```
@@ -0,0 +1,62 @@
+  workflow_dispatch:
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v4
+
+    - name: Login to Azure
+      uses: azure/login@v1
```

actionlint (1.7.7)

the runner of "azure/login@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)